lib/redis/namespace.rb

```diff
@@ -137,7 +137,7 @@ class Namespace
       "rpoplpush"        => [ :all ],
       "rpush"            => [ :first ],
       "rpushx"           => [ :first ],
-      "sadd"             => [ :first ],
+      "sadd?"            => [ :first ],
```

Comment:
why remove `sadd`? it's still a valid command

Reply:
See the mentioned issue about the deprecation warnings. It was filling our logs for this one, so this is just a suggestion to update it. Other than that I'm not really familiar with this gem TBH.

Reply:
I can indeed see the case for having both `sadd` and `sadd?`, like was done in this one: https://github.com/resque/redis-namespace/pull/171/files